python_ta/util/autoformat.py

```diff
@@ -0,0 +1,26 @@
+"""
+This file contains a function for running the Black formatting tool.
+"""
+
+import subprocess
+import sys
+from typing import Optional
+
+
+def run_autoformat(file_path: str, autoformat_options: list[str], max_linelen: Optional[int]):
+    """
```

Comment:
In general, a short description should be written on the same line as the opening """. Then a longer description (if necessary) can follow below.